staging/src/k8s.io/csi-api/pkg/apis/csi/v1alpha1/types.go

```diff
@@ -91,18 +101,56 @@ type CSINodeInfo struct {
 	// metadata.name must be the Kubernetes node name.
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	// List of CSI drivers running on the node and their properties.
-	// +patchMergeKey=driver
+	// spec is the specification of CSINodeInfo
+	// +patchMergeKey=spec
+	// +patchStrategy=merge
```

Comment:
These two directives belong with the spec.drivers field, not here. Also the key is "name" not spec.

Sorry for not noticing in the last round, I was multitasking.